content/en/dora_metrics/deployments/_index.md

```diff
@@ -33,8 +33,7 @@ The DORA Metrics private beta is closed. Fill out the form below to be added to
 
 ## Overview
 
-Deployment events are used to compute Deployment Frequency, Change Lead Time, and Change Failure Rate.
-To send deployment events, use the [DORA Metrics API][1] or the [`datadog-ci dora deployment`][2] command. The following attributes are required:
+To send your own deployment events, use the [DORA Metrics API][1] or the [`datadog-ci dora deployment`][2] command. The following attributes are required:
```

Comment:
should we mention in the overview that users can either send deployments via the API or APM Deployments tracking?